bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/To2015B.scala

```diff
@@ -116,6 +194,36 @@ object To2015B {
     ps
 
   }
+
+  // Proper Motion conversion to degrees/yr
```

Comment:
In the new model my intent was to standardize on mas/yr but this isn't documented. I'll add a doc comment.